README.md

```diff
@@ -1,134 +1,61 @@
+
+## Why use {laminr}?
+
+LaminDB offers a unique approach to data management in bioinformatics, providing:
+
+* **Unified Data and Metadata Handling**: Organize your data and its associated metadata in a structured and accessible manner.
+* **Powerful Querying and Search**: Effortlessly filter and retrieve specific data and metadata using intuitive query functions.
+* **Data Lineage Tracking**: Maintain a comprehensive history of your data transformations, ensuring reproducibility and transparency.
+* **Ontology Integration**: Leverage public ontologies (e.g., for genes, proteins, cell types) for standardized metadata annotation.
+* **Data Validation and Standardization**: Ensure data quality and consistency with built-in validation and standardization tools.
 
-## Setup
+**{laminr}** brings all these benefits to your R workflow, allowing you to seamlessly integrate LaminDB into your existing analysis pipelines.
 
-Install the development version from GitHub:
+## Installation
+
+Get started with **{laminr}** by installing the development version directly from GitHub:
 
 ``` r
 # install.packages("remotes")
 remotes::install_github("laminlabs/laminr")
 ```
 
-To install all suggested dependencies required for some functionality,
-use:
+To include all suggested dependencies for enhanced functionality, use:
 
 ``` r
 remotes::install_github("laminlabs/laminr", dependencies = TRUE)
 ```
 
-You will also need to install `lamindb`:
+You will also need to install the `lamindb` Python package:
 
 ``` bash
 pip install lamindb[aws]
 ```
 
-## Connect to an instance
-
-To connect to a LaminDB instance, you will first need to run
-`lamin login` OR `lamin load <instance>` in the terminal. This will
-create a directory in your home directory called `.lamin` with the
-necessary credentials.
-
-``` bash
-lamin login
-lamin connect laminlabs/cellxgene
-```
-
-Then, you can connect to the instance using the `laminr::connect()`
-function:
-
-``` r
-library(laminr)
-
-db <- connect("laminlabs/cellxgene")
-db
-```
-
-    cellxgene
-      Core registries
-        $Run
-        $User
-        $Param
-        $ULabel
-        $Feature
-        $Storage
-        $Artifact
-        $Transform
-        $Collection
-        $FeatureSet
-        $ParamValue
-        $FeatureValue
-      Additional modules
-        bionty
-
-## Query the instance
-
-You can use the `db` object to query the instance:
-
-``` r
-artifact <- db$Artifact$get("KBW89Mf7IGcekja2hADu")
-```
-
-You can print the record:
+## Getting started
 
-``` r
-artifact
-```
+The best way to get started with **{laminr}** is to explore the package vignettes:
```

Comment:
These will be link on the website but not the GitHub README. Maybe we can add something like "online at..." with a link?

Reply:
good idea!